docs/clients/99_graphql/index.rst

```diff
@@ -36,6 +36,20 @@ Pointing your browser to ``http://127.0.0.1:8888/explore``
 will bring up a `GraphiQL`_ interface to EdgeDB. This interface can be
 used to try out queries and explore the GraphQL capabilities.
 
+
+Known Limitations
+=================
+
+- Due to the differences between EdgeQL and GraphQL syntax
+  :eql:type:`enum <std::enum>` types which have values that use
+  symbols not found in identifiers (e.g. ``'N/A'`` or ``'NOT
+  APPLICABLE'``) cannot be properly reflected into GraphQL enums.
```

Comment:
Can we use GraphQL strings for EgdeQL enums?

Reply:
Well, technically yes, we could simply always reflect enums as GraphQL strings. Without making use of GraphQL enum type at all. On the other hand if we only sometimes reflect enums as strings that would be a bit confusing. 

We can probably even have custom types for each "enum-reflected-as-string" in GraphQL so that we can at least provide the list of valid values in the description there. But I'm not sure if that's the right approach.